packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/SubjectGroupViewer.js

```diff
@@ -54,15 +85,22 @@ const SubjectGroupViewer = forwardRef(function SubjectGroupViewer(props, ref) {
     
   const annotationMode = interactionMode === 'annotate' && isCurrentTaskValidForAnnotation
   
+  console.log('+++ C ', gridMaxWidth, gridMaxHeight)
```

Comment:
Leftover console log?

Reply:
Yikes, it is! Thank you for spotting this.